### PR TITLE
Readme motivation

### PR DESCRIPTION
**Pull request by schrodit**
**How to categorize this PR?**

/area documentation
/priority normal

**What this PR does / why we need it**:

Short Initial readme text

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,22 @@
```
Comment (MrBatschner) [inline README.md:11]: Can we put these warnings to the end? Not really inviting if the first piece of Info we see is something like "WARNING, UNFINISHED, will spoil your milk if you use it..."

File: README.md [hunk 2]
```diff
@@ -11,14 +11,22 @@
...
+
+### Start Reading
+- The documentation can be found [here](docs/README.md).
+- A list of available deployers is maintained [here](docs/deployer).
+- A glossary can be found [here](docs/concepts/Glossary.md)
 
 ### Quick Start
 
-The Landscaper is a kubernetes controller that can be easily installed via helm using the helm chart in [charts/landscaper](charts/landscaper).
+The Landscaper is a Kubernetes controller that can be easily installed via helm using the helm chart in [charts/landscaper](charts/landscaper).
```
Comment (MrBatschner) [inline README.md:29]: "Drop the 'the'."

**Review by achimweigel — APPROVED** [review]
Looks good for me

**Review by MrBatschner — APPROVED** [review]

**Pull request merged**